### DISCUSSION TITLE
Pulse and Superbase not works

### DISCUSSION THREAD

**Discussion by joaoeudes7** [prisma/prisma#23261] [Q&A]
### Question

On try set the database uri receives the error:

`The specified datasource is unreachable. The error was: publication "prisma_pulse_818e7a70" already exists. This could be caused by an incorrect datasource URL or database firewall rules. This is probably not a problem with Pulse itself.`

This is the same uri used to enable the accelerate, but to the Pulse dont works

### Expected behavior (optional)

Enable Pulse

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by nurul3101** [comment]
Hey @joaoeudes7 👋 

Are you using the default connection string provided by Supabase?
Can you share the format of the connection string you used (after obfuscating username and password)?

**Reply by joaoeudes7** [reply]
postgres://postgres.[project]:[pass]@[host]:5432/postgres?schema=public

**Reply by ludralph** [reply]
Hi @joaoeudes7 

Can you please let us know if this issue is resolved for you?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
